### PR TITLE
Highlight my vote even if it was made on another device

### PR DESCRIPTION
**Pull request by andybalaam**
---
This change is marked as an *internal change* (Task), so will not be included in the changelog.

Preview: https://61a0a702f2428631a8a14fd3--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/components/views/messages/MPollBody.tsx [hunk 1]
```diff
@@ -171,15 +163,18 @@ export default class MPollBody extends React.Component<IBodyProps, IState> {
         }
 
         const pollId = this.props.mxEvent.getId();
-        const votes = this.collectVotes();
+        const userVotes = this.collectUserVotes();
+        const votes = countVotes(userVotes, this.props.mxEvent.getContent());
         const totalVotes = this.totalVotes(votes);
+        const userId = MatrixClientPeg.get().getUserId();
```
Comment (t3chguy) [inline src/components/views/messages/MPollBody.tsx:169]: can we use a MatrixClientContext here instead of MatrixClientPeg?

**Review by t3chguy — APPROVED** [review]
LGTM otherwise

File: src/components/views/messages/MPollBody.tsx (hunk 1 above)
Comment (andybalaam) [reply src/components/views/messages/MPollBody.tsx]: Something like 17b726cfb0 ?  I've never used this before so please check I did it right :-)

Comment (t3chguy) [reply src/components/views/messages/MPollBody.tsx]: Yup! This way will make any future transitions to a multi-account world much easier, thanks!

**Pull request merged**